libclc/libspirv/include/libspirv/misc/shuffle2.h

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#define __CLC_FUNCTION __spirv_ocl_shuffle2
+
+// Integer-type decls
+#define __CLC_BODY <clc/misc/shuffle_decl.inc>
```

Comment:
As with upstream, this should be `shuffle2_decl`

Reply:
done, thanks